src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,25 @@
-export const CommentList = () => (
-  <>Put the list here</>
-);
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export function CommentList({ comments }) {
+  const hasComments = !!comments.length;
```

Comment:
it's gonna be automatically converted to boolean if you use ternary operator, !! is useful in case when you use &&